docker_mcp/server.py

```
@@ -1765,18 +1472,21 @@ def create_app():
+
 def get_app():
     """Get the FastMCP app instance, creating it if necessary."""
-    global _app_instance
+    global _app_instance  # noqa: PLW0603
     if _app_instance is None:
         _app_instance = create_app()
     return _app_instance
 
+
 # Direct access for FastMCP (creates app on first access)
 def __getattr__(name):
```

Ruff (0.14.3)

Missing return type annotation for private function `__getattr__`

(ANN202)